test/console-smoke/console.ts

```diff
@@ -0,0 +1,99 @@
+import { By, WebDriver } from 'selenium-webdriver'
+import {
+  clickElement,
+  clickWebElement,
+  getByDataTestID,
+  getElementText,
+  getElements,
+  getWebElementText,
+  isElementDisplayed,
+  sendKeysToElement,
+  waitForElementToBeReady,
+  waitForElementToDisappear
+} from '../e2e/helpers/selenium-util'
+import { elementIsDisabled } from 'selenium-webdriver/lib/until'
+
+export class Console {
+  private readonly connect: By = getByDataTestID('connect-vega-wallet')
+  private readonly connectToBrowserWallet: By = getByDataTestID('connector-injected')
+  private readonly orderTab: By = getByDataTestID('Order')
+  private readonly networkDropdown: By = getByDataTestID('navbar-network-switcher-trigger')
+  private readonly orderSize: By = getByDataTestID('order-size')
+  private readonly orderPrice: By = getByDataTestID('order-price')
+  private readonly placeOrder: By = getByDataTestID('place-order')
+  private readonly welcomeDialog: By = getByDataTestID('dialog-close')
+  private readonly connectDialog: By = getByDataTestID('wallet-dialog-title')
+  private readonly successToast: By = By.css("[data-testid='toast'][class*='bg-vega-green']")
+  private readonly closeAnyToast: By = By.css("[data-testid='toast-close']")
+  private readonly marketsParentElement: By = By.css("[data-testid*='pathname-/markets']")
+  private readonly marketSelector: By = getByDataTestID('popover-trigger')
+  private readonly marketSelectorPrice: By = By.css("[data-testid='market-selector-price']")
+
+  constructor(private readonly driver: WebDriver) {}
+
+  async checkTransactionSuccess() {
+    expect(await isElementDisplayed(this.driver, this.successToast), 'expected to see a success toast', {
+      showPrefix: false
+    }).toBe(true)
+  }
+
+  async connectToWallet() {
+    console.log('clicking connect')
+    await clickElement(this.driver, this.connect)
+    console.log('clicked connect')
+    expect(
+      await isElementDisplayed(this.driver, this.connectToBrowserWallet),
+      'expected to see the connect to browser wallet button',
+      { showPrefix: false }
+    ).toBe(true)
+
+    await clickElement(this.driver, this.connectToBrowserWallet)
+  }
+
+  async waitForConnectDialogToDissapear() {
+    await waitForElementToDisappear(this.driver, this.connectDialog)
+  }
+
+  async goToOrderTab() {
+    if (await isElementDisplayed(this.driver, this.placeOrder, 3)) {
+      return
+    }
+    await clickElement(this.driver, this.orderTab)
+  }
+
+  async selectTBTCMarket() {
+    const marketParentElement = await waitForElementToBeReady(this.driver, this.marketsParentElement)
+    await clickWebElement(this.driver, marketParentElement.findElement(this.marketSelector))
+    const markets = await getElements(this.driver, this.marketSelectorPrice)
+    for (const market of markets) {
+      if ((await getWebElementText(this.driver, market)).includes('tBTC')) {
```

Comment:
Just wondering if we need to be concerned about different asset classes with this check... I think it's probably fine for now and let's see if it breaks rather than being super specific in terms of our market selection. Let's just keep in mind.

Reply:
I figured this was ok and less effort than making sure the wallet has balance for each asset. If it is a pain then I will top of the wallet so it doesn't matter which one